src/plugins/saved_objects_management/public/management_section/objects_table/components/delete_confirm_modal.tsx

```diff
@@ -97,13 +100,25 @@ export const DeleteConfirmModal: FC<DeleteConfirmModalProps> = ({
               iconType="warning"
               color="warning"
             >
-              <p>
-                <FormattedMessage
-                  id="savedObjectsManagement.objectsTable.deleteConfirmModal.cannotDeleteCallout.content"
-                  defaultMessage="{objectCount, plural, one {# object is} other {# objects are}} hidden and cannot be deleted. {objectCount, plural, one {It was} other {They were}} excluded from the table summary."
-                  values={{ objectCount: undeletableObjects.length }}
-                />
-              </p>
+              {hiddenObjects.length > 0 && (
+                <p>
+                  <FormattedMessage
+                    id="savedObjectsManagement.objectsTable.deleteConfirmModal.cannotDeleteCallout.content"
+                    defaultMessage="{objectCount, plural, one {# object is} other {# objects are}} hidden and cannot be deleted. {objectCount, plural, one {It was} other {They were}} excluded from the table summary."
```

Comment:
I know this is pre-existing, but seeing "excluded from table summary" text on one line looks a bit weird:

<img width="868" alt="Screenshot 2024-02-08 at 12 34 23" src="https://github.com/elastic/kibana/assets/8155004/928d6bef-1934-4e69-aeb4-41289803a95e">

I think we should remove it and change the callout title to to something like:

<img width="829" alt="Screenshot 2024-02-08 at 12 36 43" src="https://github.com/elastic/kibana/assets/8155004/bd6652aa-1643-4318-a4dd-25500bf28c06">

WDYT?

Reply:
This makes sense to me